src/main/resources/configCivcraft.yml

```diff
@@ -7159,10 +7159,10 @@ recipes:
 #        amount: 160
       inksacs:
         material: INK_SACK
-        amount: 64
+        amount: 32
       apples:
         material: APPLE
-        amount: 32
+        amount: 16
```

Comment:
This recipe is a huge mess. I don't think anyone would ever run it over the other two in its current state. Just needs to be changed in general. Apples are fine if you didn't already need 2 different kind of tree farms.